README.md

```diff
@@ -138,12 +138,18 @@ The prompt shows information you need while you're working, while staying sleek
 
    #### Install Latest Version
 
-   ##### With Shell:
```

Comment:
I think this should remain "With Shell", so we can add a "With PowerShell" heading when #690 is completed.

Reply:
How about `Prebuilt Binary, With Shell`?

Reply:
`Prebuilt binary with Shell` works for me. 👍